tests/lib/snaps/test-snapd-service/meta/snap.yaml

```diff
@@ -5,4 +5,7 @@ apps:
   command: bin/start
   daemon: simple
   reload-command: bin/reload
+ test-snapd-other-service:
```

Comment:
Can we please take this chance to fix the indentation here (4 spaces per indent on all shell, yaml, python).